pkg/config/config.go

```diff
@@ -33,22 +40,67 @@ func GetConfigs(filename string) (*Configs, error) {
 		log.Fatal(err)
 	}
 
+	if err := parseEnv(&configs); err != nil {
+		log.Fatal(err)
+	}
+
 	return &configs, nil
 }
 
-// GetAllImages extracts a set of images required for all the tasks to run
-func (c *Configs) GetAllImages() ([]string, error) {
-	var images []string
-	var set = make(map[string]bool)
+func parseEnv(configs *Configs) error {
+	file := viper.GetString("DotenvFile")
+	envs, err := godotenv.Read(file)
+	if err != nil {
+		log.Warn(err)
+	}
 
-	for _, tasks := range c.Tasks {
-		for _, task := range tasks {
-			set[task.Image] = true
+	for k, tasks := range (*configs).Tasks {
+		for j, task := range tasks {
+			for i, envVar := range task.Envs {
+				var str = strings.Split(envVar, "=")
+				if len(str) != 2 {
+					return fmt.Errorf(
+						`config: invalid format of environment variable: %v`,
+						envVar,
+					)
+				}
+				var pattern = "^`\\$.+`$"
```

Comment:
Shouldn't the RegEx be`"^`\$.+`$"` or I'm mistaken?

Reply:
No, the suggested regex produces an error. See the working on golang playground [here](https://play.golang.org/p/jCqEzJMJ_23).